.tito/cockpit-build.sh

```diff
@@ -8,13 +8,8 @@ if [ -n "${USE_SUSE}" ] ; then
 fi
 
 echo "Generating cockpit dist-gzip..."
-if [ ! $(which yarn) ] ; then
-  echo "Unable to build cockpit, yarn not installed (see README)"
```

Comment:
I don't know if you want this same nice message for npm itself.  You will get a 'npm command not found' error anyway...

Reply:
Yea that is fine.